lib/ace/snippets.js

```diff
@@ -894,8 +894,9 @@ var TabstopManager = function(editor) {
             ed.tabstopManager.detach();
         },
         "Return": function(ed) {
-            //ed.tabstopManager.tabNext(1);
-            return false;
+	    //for papeeria autocomplete
```

Comment:
Yes.
It's for fix bug with multiply caret with \begin{env}.
After "enter" we had 2 caret